frame/election-provider-support/Cargo.toml

```diff
@@ -14,6 +14,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "0.9.0", default-features = false, features = ["derive"] }
```

Comment:
Missing `scale-info/std` for this Cargo.toml?

Reply:
Actually `scale-info` isn't even used here, removed